nav2_map_server/src/map_server.cpp

```diff
@@ -1,29 +1,62 @@
-// Copyright (c) 2018 Intel Corporation
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+/* Copyright (c) 2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Copyright 2019 Rover Robotics
```

Comment:
Yes, this file became to be dual-licensed because of `occ_grid_loader.cpp` (licensed under BSD-3 by Rover Robotics, Brian Gerkey and Willow Garage, Inc.) was merged with `map_server.cpp` (licensed under Apache 2.0 by Intel Corporation). There is some code from OccGridLoader (e.g. `OccGridLoader::on_configure()` was moved almost w/o changes), so I am not sure how can we skip one or another license. Is there an issue to license this file under dual-licensing (Apache 2.0 + BSD-3-Clause)?

Reply:
I don’t think Intel had the right to relicense just one file in a BSD project to Apache, I think that was just a mistake of boilerplate. @crdelsey thoughts?